tests/unit/policyEvaluator.js

```diff
@@ -1161,6 +1161,74 @@ describe('policyEvaluator', () => {
                 };
                 check(requestContext, rcModifiers, policy, 'Neutral');
             });
+
+            it('should allow with StringEquals operator and ExistingObjectTag ' +
```

Comment:
I'd suggest adding a couple failing tests for unsupported patterns like `s3:FooBar/tagKey` (see earlier comment)

Reply:
oh should've read this comment before responding to the last one. makes sense now 👍